app/Resources/views/includes/base.html.twig

```diff
@@ -12,14 +12,20 @@
         <!-- Optional theme -->
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
 
-        <!-- Latest compiled and minified JavaScript -->
-        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
-
-
         <link rel="icon" type="image/x-icon" href="{{ asset('favicon.ico') }}" />
+        <link rel="stylesheet" href="{{ asset('css/base.css') }}">
     </head>
     <body>
         {% block body %}{% endblock %}
-        {% block javascripts %}{% endblock %}
+        {% block javascripts %}
```

Comment:
Was bringt der {% block javascripts %} TWIG Block? Kann der JS-Teil nicht hard in den Quelltext gecodet werden?

Reply:
Nein, denn wenn andere Seiten noch Javascript hinzufügen wollen, können sie es durch diesen Block an dieser Stelle tun

Reply:
Du bist gerade im Besitz des Tickets #38 . Ist das Feedback.js dort richtig eingebunden?